cmd/keycloakb/keycloak_bridge.go

```diff
@@ -122,17 +122,17 @@ func main() {
 		influxWriteInterval = c.GetDuration("influx-write-interval")
 
 		// DB - for the moment used just for audit events
-		auditRwDbParams = database.GetDbConfig(c, "db-audit-rw", !c.GetBool("events-db"))
+		auditRwDbParams = database.GetDbConfig(c, "db-audit-rw")
 
 		// DB - Read only user for audit events
-		auditRoDbParams = database.GetDbConfig(c, "db-audit-ro", false)
+		auditRoDbParams = database.GetDbConfig(c, "db-audit-ro")
 
 		// DB for custom configuration
-		configRwDbParams = database.GetDbConfig(c, "db-config-rw", !c.GetBool("config-db-rw"))
```

Comment:
for example, config-db-rw is still set at some point in this file. I imagine we should remove it ? (same for the others)

Reply:
Ok... I just renamed fields when initializing the configuration.